settings/settings.py.j2

```diff
@@ -209,6 +209,9 @@ LOGGING_CONFIG = None
 # Disable EZID's logging config (would be performed in impl.log)
 LOGGING_CONFIG_FILE = None
 
+# Default log_level to DEBUG if not specified.
+{% if log_level is not defined %}{% set log_level = 'DEBUG' %}{% endif %}
```

Comment:
We have a "log_level" parameter set AWS SSM as /uc3/ezid/ui/default/log_level. We use {{log_level}} to get the value in thesettings.py.j2. Are these two variables refer to the same thing?